### PR TITLE
Filter component

### PR DESCRIPTION
**Pull request by 0figueiredo**
-Split filtercomponent.js in two separate windows.

### CODE REVIEW

File: src/Components/FilterComponents/FilterPages.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import SelectCategorie from './SelectCategorie';
+import SelectFirstImportancePoi from './SelectFirstImportancePoi';
+import SelectSecondImportancePoi from './SelectSecondImportancePoi';
+import '../ComponentsCSS/createPoiForm.scss';
+
+const mapStateToProps = state => ({
+  page: state,
```
Comment (erabin05) [inline src/Components/FilterComponents/FilterPages.js]: 'page' is equal to state ??

File: src/Components/FilterComponents/SelectFirstImportancePoi.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+const mapStateToProps = state => ({
+  poiKeywordsDisplay: state.pois.poiKeywordsDisplay,
+  displaySecondImportancePoiPage: state.pois.displaySecondImportancePoiPage,
+  firstIndicationIsDisplayed: state.pois.firstIndicationIsDisplayed,
+  filterKeywordPageDisplay: state.pois.filterKeywordPageDisplay,
+});
+
+const selectRightKeywordChildren = (specificKeyword, poiKeywordsDisplay) => poiKeywordsDisplay.filter(keyword => keyword.parent_id === specificKeyword.id);
+
+const onlyKeywordsFirstImportance = keywords => keywords.filter(
```
Comment (erabin05) [inline src/Components/FilterComponents/SelectFirstImportancePoi.js:18]: Indent 'selectRightKeywordChildren' and 'onlyKeywordsFirstImportance' the same way

**Review by erabin05 — CHANGES_REQUESTED** [review]
- in FilterPages.js, why in the mapStateToProps one of the props in equal to state ?
- Indentation can be improve

File: src/Components/FilterComponents/FilterPages.js (hunk 1 above)
Comment (0figueiredo) [reply src/Components/FilterComponents/FilterPages.js]: file removed

File: src/Components/FilterComponents/SelectFirstImportancePoi.js (hunk 2 above)
Comment (0figueiredo) [reply src/Components/FilterComponents/SelectFirstImportancePoi.js:18]: indentation changed

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**